### PR TITLE
LPS-146772 Small refactor for dynamic code

### PR DESCRIPTION
**Pull request by luisxx123**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Employees.js [hunk 1]
```diff
@@ -46,102 +55,168 @@ export function FormEmployees({form}) {
...
-				rules={{required: 'This field is required'}}
-			/>
+						name={setFormPath('hasFein')}
+						rules={{required: true}}
+					/>
+				</div>
+
+				<FEINControlledInput
+					control={control}
+					label="Federal Employer Identification Number (FEIN)"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Employees.js:99]: Create or use a variable for that label.

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Employees.js [hunk 2]
```diff
@@ -46,102 +55,168 @@ export function FormEmployees({form}) {
...
+
+			<MobileContainer
+				{...mobileContainerProps}
+				mobileSubSection={getMobileSubSection(
+					SUBSECTION_KEYS.BUSINESS_YEAR_OPERATION
+				)}
+			>
+				<ControlledSwitch
+					control={control}
+					label="Does your business operate year round?"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Employees.js:141]: Create or use a variable for that label.

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Employees.js [hunk 3]
```diff
@@ -46,102 +55,168 @@ export function FormEmployees({form}) {
...
+
+			<MobileContainer
+				{...mobileContainerProps}
+				mobileSubSection={getMobileSubSection(
+					SUBSECTION_KEYS.EMPLOYEES_AMOUNT
+				)}
+			>
+				<NumberControlledInput
+					control={control}
+					label="How many full or part time employees do you have?"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Employees.js:156]: Create or use a variable for that label.

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Employees.js [hunk 4]
```diff
@@ -46,102 +55,168 @@ export function FormEmployees({form}) {
...
+
+			<MobileContainer
+				{...mobileContainerProps}
+				mobileSubSection={getMobileSubSection(
+					SUBSECTION_KEYS.OWNERS_ANNUAL_PAYROLL
+				)}
+			>
+				<CurrencyControlledInput
+					control={control}
+					label="What do you anticipate your annual payroll will be for all owner(s) over the next 12 months?"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Employees.js:201]: Create or use a variable for that label.

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Employees.js [hunk 5]
```diff
@@ -46,102 +55,168 @@ export function FormEmployees({form}) {
...
+
+			<MobileContainer
+				{...mobileContainerProps}
+				mobileSubSection={getMobileSubSection(
+					SUBSECTION_KEYS.EMPLOYEES_ANNUAL_PAYROLL
+				)}
+			>
+				<CurrencyControlledInput
+					control={control}
+					label="What do you anticipate your annual payroll will be for all employees over the next 12 months?"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/components/containers/Forms/Employees.js:215]: Create or use a variable for that label.

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/utils/constants.js [hunk 6]
```diff
@@ -26,20 +26,31 @@ export const TOTAL_OF_FIELD = {
 };
 
 export const SUBSECTION_KEYS = {
+	BUSINESS_ANUAL_GROSS_REVENUE:
+		'What is your estimated annual gross revenue for the next 12 months?',
 	BUSINESS_EMAIL: 'Business Email',
+	BUSINESS_FEDERAL_EMPLOYER_ID:
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/get-a-quote/utils/constants.js:32]: That is not an ID. I think that you could improve the name of this constant.

**Review by silasbsi — CHANGES_REQUESTED** [review]
LGTM :fire: But needs some adjustments

**Review by silasbsi — APPROVED** [review]

**Comment by silasbsi** [comment]
LGTM :fire: @kevenleone could you make another review?

**Review by kevenleone — APPROVED** [review]
It's looks good!

GJ @Mylena-Rodrigues and @luisxx123

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**